src/index.html

```diff
@@ -17,11 +18,408 @@
+            <a
+              href="#"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <div class="main">
+      <!-- #region card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h1 class="card__code">Product code: 195434</h1>
+
+        <div class="card__active">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <h1 class="card__reviews">Reviews: 5</h1>
+        </div>
+
+        <div class="card__value">
+          <h1 class="card__price">Price:</h1>
+          <h1 class="card__price-number">$2,199</h1>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+      <!-- #endregion -->
+
+      <!-- #region card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h1 class="card__code">Product code: 195434</h1>
+
+        <div class="card__active">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <h1 class="card__reviews">Reviews: 5</h1>
+        </div>
+
+        <div class="card__value">
+          <h1 class="card__price">Price:</h1>
+          <h1 class="card__price-number">$2,199</h1>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+      <!-- #endregion -->
+
+      <!-- #region card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
```

Comment:
```suggestion
          src="./images/imac.jpeg"
```
should work

Reply:
don't forget to fix it
<img width="622" alt="image" src="https://github.com/user-attachments/assets/191d52e6-9b97-4b70-9379-2d0089b15fc4">